README.md

```diff
@@ -29,70 +37,178 @@ Add UIImageColors to your `Cartfile`:
 github "jathu/UIImageColors"
 ```
 
-## Example
-
-Asynchronous example:
-
-```swift
-let image = UIImage(named: "yeezus.png")
-
-image.getColors { colors in
-  backgroundView.backgroundColor = colors.background
-  mainLabel.textColor = colors.primary
-  secondaryLabel.textColor = colors.secondary
-  detailLabel.textColor = colors.detail
-}
-```
-
-Synchronous example:
-
-```swift
-let colors = UIImage(named: "yeezus.png").getColors()
-
-backgroundView.backgroundColor = colors.background
-mainLabel.textColor = colors.primary
-secondaryLabel.textColor = colors.secondary
-detailLabel.textColor = colors.detail
-```
-
-## Image Methods
-
-```swift
-getColors() -> UIImageColors?
-getColors(quality: ImageColorsQuality) -> UIImageColors?
-getColors(_ completion: (UIImageColors?) -> Void) -> Void
-getColors(quality: UIImageColorsQuality, _ completion: (UIImageColors?) -> Void) -> Void
-```
-
-## UIImageColors Objects
+### Manual
 
-`UIImageColors` is struct that contains four different `UIColor` (or `NSColor` on macOS) variables.
+Copy the [UIImageColors](/Sources/UIImageColors) folder into your project.
+
+For Objective-C projects copy the entire [Sources](/Sources) folder.
+
+## Examples
+
+<details>
+  <summary>iOS (Swift)</summary>
+  <br>
+  
+  ```swift
+  import UIImageColors
+  ```
+  
+  Synchronous:
+  
+  ```swift
+  let image = UIImage(named: "example.png")
+  let colors = image.getColors()
+  // use colors
+  ```
+  
+  Asynchronous(completion-handler):
+  
+  ```swift
+  let image = UIImage(named: "example.png"
+  image.getColors { colors in
+      // use colors
+  }
+  ```
+  
+  Asynchronous(async/await):
+  
+  ```swift
+  let image = UIImage(named: "example.png")
+  let colors = await image.colors()
+  // use colors
+  ```
```

Comment:
We can condense this.

```suggestion
import UIImageColors

let image = UIImage(named: "example.png")

// Synchronous
let colors = image.getColors()

// Async/await
let colors = await image.colors()

// Asynchronous (completion handler)
image.getColors { colors in
    // use colors...
}
```

Let's do something similar for the other examples

Reply:
You'r right, thats better

Reply:
Done